<launch>
    <!-- Load parameters from YAML -->
    <rosparam file="$(find msd700_webui_control)/config/particle_align_params.yaml" command="load" />

    <!-- Launch the Particle Align Validator node -->
    <node pkg="msd700_webui_control" type="particle_align_validator.py" name="particle_align_validator" output="screen" />
</launch>
